virttest/qemu_vm.py

```diff
@@ -1340,6 +1340,10 @@ def sort_key(dev):
             for pcic in params.objects("pci_controllers"):
                 dev = devices.pcic_by_params(pcic, params.object_params(pcic))
                 pcics.append(dev)
+            extra_root_port = params.get('extra_root_port', '0')
+            for num in range(int(extra_root_port)):
+                pcics.append(devices.pcic_by_params("pcie_root_port_%s" % num,
+                                                    {"type": "pcie-root-port"}))
```

Comment:
For now hardcoding `pcie-root-port` is probably fine and in the future we might introduce `extra_root_port_type`. So no issues here...